### PR TITLE
242 supporting facts stage 1

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/m5Auuse3/242-supporting-facts-actions-i-can-take-stage-ii

## Description

- Adds a grid of links to the help-self page
- As some common things are useful, grid styles abstracted to theme

@neontribe/sea-map

### CODE REVIEW

File: src/Page/HelpSelf.elm [hunk 1]
```diff
@@ -31,10 +34,59 @@ update msg model =
...
         ]
+
+
+infoLink : String -> String -> Html msg
+infoLink title link =
+    li
+        [ css
+            [ cardStyle
+            , withMedia [ only screen [ Media.minWidth (px 576) ] ]
+                [ flex3 zero zero threeColumn ]
```
Comment (katjam) [inline src/Page/HelpSelf.elm:61]: Do these get added in addition to the cardStyles?

I wonder if would be clearer if the         `, flex3 zero zero twoColumn` was excluded from the cardStyles batch and added as an either or?

Happy to go with this though if it works and that is how the media thing is supposed to be constructed.

File: src/Page/HelpSelf.elm [hunk 2]
```diff
@@ -31,10 +34,59 @@ update msg model =
...
+        , Css.height (pct 100)
+        , justifyContent center
+        , textDecoration none
+        , Css.width (pct 100)
+        ]
+
+
+cardStyle : Style
+cardStyle =
+    batch
```
Comment (katjam) [inline src/Page/HelpSelf.elm:85]: I guess there might be potentially other attributes we want to adjust in the media query once the real style get up - so could be good to have `desktopCardStyle` and `mobileCardStyle`if that happens to make it more clear.

File: tests/HelpSelfTest.elm [hunk 3]
```diff
@@ -21,14 +21,7 @@ suite =
                 view {}
                     |> Html.Styled.toUnstyled
                     |> Query.fromHtml
-                    |> Query.contains [ Html.text "Help Self page - grid and singles" ]
-        , test "HelpSelf view has 3 nav links" <|
```
Comment (katjam) [inline tests/HelpSelfTest.elm:25]: Did you deliberately remove the counting links test?

**Review by katjam — COMMENTED** [review]
I think it looks ok. Few comments.

File: tests/HelpSelfTest.elm (hunk 3 above)
Comment (Nikomus) [reply tests/HelpSelfTest.elm:25]: Yes, I wasn't sure it was helpful to count the number of links at this stage. I made five blocks mostly for responsive styling, but expect that to change.

Comment (katjam) [reply tests/HelpSelfTest.elm:25]: Agree.

**Review by katjam — APPROVED** [review]
Awesome! :8ball:

**Pull request merged**